scrunch/datasets.py

```diff
@@ -2317,13 +1529,18 @@ class Variable(ReadOnly, DatasetSubvariablesMixin):
     def __init__(self, var_tuple, dataset):
         """
         :param var_tuple: A Shoji Tuple for a dataset variable
-        :param dataset: a Dataset() object instance
+        :param dataset: a Dataset object instance
         """
         self.shoji_tuple = var_tuple
         self.is_instance = False
         self._resource = None
         self.url = var_tuple.entity_url
         self.dataset = dataset
+        # for Variables with subvariables
+        try:
+            self._reload_variables()
+        except AttributeError:
```

Comment:
Why would this raise AttributeError? You were considering `.variables` and then `.subvariables` as 2nd case, if we have a 3rd case we didn't contemplate then it is programming error and should not pass silent.